paper.tex

```diff
@@ -339,7 +339,7 @@ \section{Genome ARGs}
 In many settings, nodes are dated, i.e.\ each
 node $u\in N$ is associated with a time $\tau_u$.
 A node can represent any genome along a
-% A gamete *is* one of the genomes along a chain of cell divisions? 
+% A gamete *is* one of the genomes along a chain of cell divisions?
 % I don't see what adding a gamete clarifies, and we cite Hudson plenty
 chain of cell divisions~\ref{sec-cell-lineages-and-args},
 or can be interpreted as representing one of the genomes of an
```

Comment:
We say "can be interpreted as representing one of the genomes of an individual". A gamete isn't a cell in an individual (it's its own thing), which is why I thought we should add this.